src/Query/Admin/Model/Organization.php

```diff
@@ -36,11 +34,6 @@ public function alias(): string
         return $this->alias;
     }
 
-    public function ownerEmail(): string
-    {
-        return $this->ownerEmail;
```

Comment:
for now removed, maybe we will need `members` feature for admin (but for now it's not needed)